src/lib/modules/scaffolding/description.ts

```diff
@@ -0,0 +1,106 @@
+import Ajv from "ajv";
+
+import { Logger } from "../../../typings/logger";
+import { schema } from "./schema";
+
+const fs = require("../../core/fs");
+
+const ajv = new Ajv();
+const scaffoldingSchema = ajv.compile(schema);
+
+interface Properties {
+  [propertyName: string]: string;
+}
+
+interface Data {
+  [contractName: string]: Properties;
+}
+
+const ASSOCIATIONS = ["belongsTo", "hasMany"];
+const IPFS = ["ipfsText", "ipfsImage"];
+
+export class Description {
```

Comment:
Here I also think that `Description` is a bit too generic. Can we make this a bit more "concrete"? Keep in mind that this makes searching for stuff harder as well.

Reply:
I agree with you but could not find anything better. Any suggest?